catalog/resource_catalog_workspace_binding_test.go

```diff
@@ -0,0 +1,43 @@
+package catalog
+
+import (
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/qa"
+)
+
+func TestCatalogWorkspaceBindingsCornerCases(t *testing.T) {
```

Comment:
I'm not sure why but this test is extremely flaky - It passes locally maybe 9 out of 10 time but then fails randomly like it did in this action. https://github.com/databricks/terraform-provider-databricks/actions/runs/5176590327/jobs/9326664436?pr=2364

Reply:
I don't think it's flaky, rather than it always fails the first run, and then succeeds to following runs